src/ui/Interface.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function Interface (options) {
+
+    options.latId = options.latId || 'lat';
+    options.lngId = options.lngId || 'lng';
+
+    function panMapWhenInputsChange() {
+    var lat = document.getElementById(options.latId);
```

Comment:
Can we indent these lines 2 more spaces for easier readability? And can we just run `Interface.panMapWhenInputsChange();` here instead of in BlurredLocation's constructor?